cmd/main.go

```diff
@@ -37,7 +37,7 @@ func init() {
 //
 // Setup the DB and models.
 func Setup() (db *gorm.DB, err error) {
-	err = migration.Migrate(migration.All())
+	err = migration.Migrate(migration.All(), migration.SupportedFrom)
```

Comment:
I'm curious about why this is passed in since it's managed/owned by the migration package?  The setting can be manipulated for testing.

Reply:
Also (don't mean to be picky but) I wonder if somethiing like `minVersion` would be _better_?